prover/config.go

```diff
@@ -116,8 +117,12 @@ func NewConfigFromCliContext(c *cli.Context) (*Config, error) {
 		if !c.IsSet(flags.L2NodeVersion.Name) {
 			return nil, errors.New("--prover.l2NodeVersion flag is required if guardian prover is set")
 		}
-	}
 
+		// If we are running a guardian prover, a raiko host endpoint is required.
+		if !c.IsSet(flags.RaikoHostEndpoint.Name) {
```

Comment:
We can remove this check now, since https://github.com/taikoxyz/taiko-client/pull/798